src/structures/ThreadChannel.js

```diff
@@ -234,6 +234,22 @@ class ThreadChannel extends Channel {
     return this.parent?.permissionsFor(memberOrRole) ?? null;
   }
 
+  /**
+   * Fetches the owner of this thread
+   * @param {FetchOwnerOptions} [options] The options for fetching the member
+   * @returns {Promise<ThreadMember>}
+   */
+  async fetchOwner({ cache = true, force = false } = {}) {
+    if (!force) {
+      const existing = this.cache.get(this.ownerId);
```

Comment:
What's `this.cache` inside a `ThreadChannel`? Did you mean `this.members.cache.get`?

Reply:
Oopsie yeah forgot that